.github/workflows/unit_test.yml

```diff
@@ -56,3 +56,28 @@ jobs:
           GOBIN: /home/runner/go/bin
     - name: Run 
       run: make test-verbose
+
+  integration_test:
+    runs-on: ubuntu-latest
+    needs: unit_test_without_bcc
+    steps:
+      - uses: actions/checkout@main
+      - uses: actions/setup-go@main
+        with:
+          go-version: 1.18
+      - name: install kind
+        run: go install sigs.k8s.io/kind@v0.12.0
+      - name: check version
+        run: kind version
+      - name: install kubectl
+        run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+      - name: check version
+        run: kubectl version || true
+      - name: infra up
+        run: ./scripts/kind.sh
+      - name: build image
+        run: make
+      - name: upload image to local registry
+        run: kind load docker-image quay.io/sustainable_computing_io/kepler:latest
```

Comment:
@marceloamaral has a recent change to build local images

Reply:
I suppose for this PR we can ignore the change for local image build?
the logic:
use local docker to build an image with tag `quay.io/sustainable_computing_io/kepler:latest` and upload to local registry here.

Reply:
as all image files at github agent locally, hence I support no need to worry too much?

Reply:
resolve as out of date